test/new-e2e/tests/updater/linux_test.go

```diff
@@ -70,6 +70,10 @@ func TestDebianX86(t *testing.T) {
 	runTest(t, "dpkg", os.AMD64Arch, os.UbuntuDefault)
 }
 
+func TestSuse(t *testing.T) {
+	runTest(t, "rpm", os.AMD64Arch, os.SuseDefault)
+}
```

Comment:
adding a arm64 too would be great to ensure 100% coverage of each package

Reply:
indeed, done